content/docs/forms.md

```diff
@@ -21,15 +21,15 @@ HTML form elements work a little bit differently from other DOM elements in Reac
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
+ऊपर दिया गया फॉर्म अन्य HTML फॉर्म की तरह सबमिट करने पर नए पेज पर चला जाता है। यदि आप इस व्यवहार को React में चाहते हैं, तो यह काम करता है। लेकिन ज्यादातर मामलों में, जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है क्यूंकि वह फॉर्म को सब्मिट करने के साथ साथ, उसकी उस डाटा तक भी पहुँच होती है जो यूजर फॉर्म मैं भरता है। इसे प्राप्त करने का मानक तरीका "कंट्रोल्ड कौम्पोनॅन्ट" नामक एक तकनीक के साथ है।
```

Comment:
लेकिन ज्यादातर मामलों में, जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है क्यूंकि वह फॉर्म को सब्मिट करने के साथ साथ, उसकी उस डाटा तक भी पहुँच होती है जो यूजर फॉर्म मैं भरता है। इसे प्राप्त करने का मानक तरीका "कंट्रोल्ड कौम्पोनॅन्ट" नामक एक तकनीक के साथ है।

this changes the meaning of the original sentence, reword both the statements a bit to match the original ?

Reply:
Have tried one approach to it in next commit. Discussed with several people on their understanding and thus modified it.

Reply:
इस तरह के व्यवहार को प्राप्त करने का एक मानक तरीका है "कंट्रोल्ड कौम्पोनॅन्ट" नामक तकनीक। could be reworded to

इस तरह के व्यवहार को प्राप्त करने का एक मानक तरीका "कंट्रोल्ड कौम्पोनॅन्ट" नामक तकनीक है।

rest everything is good